packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Branding/index.tsx

```diff
@@ -149,32 +165,76 @@ function Branding({ application, isActive }: Props) {
+                        </FormField>
+                      )}
+                    />
+                    <Controller
+                      control={control}
+                      name="color.darkPrimaryColor"
+                      render={({ field: { name, value, onChange } }) => (
+                        <FormField title="application_details.branding.brand_color_dark">
+                          <ColorPicker name={name} value={value} onChange={onChange} />
+                        </FormField>
+                      )}
+                    />
+                  </>
+                )}
+              </div>
+            )}
           </FormCard>
-          <FormCard
-            title="application_details.branding.more_info"
-            description="application_details.branding.more_info_description"
-          >
-            <FormField title="application_details.branding.terms_of_use_url">
-              <TextInput
-                {...register('termsOfUseUrl', {
-                  validate: (value) =>
-                    !value || uriValidator(value) || t('errors.invalid_uri_format'),
-                })}
-                error={errors.termsOfUseUrl?.message}
-                placeholder="https://"
-              />
-            </FormField>
-            <FormField title="application_details.branding.privacy_policy_url">
-              <TextInput
-                {...register('privacyPolicyUrl', {
-                  validate: (value) =>
-                    !value || uriValidator(value) || t('errors.invalid_uri_format'),
-                })}
-                error={errors.privacyPolicyUrl?.message}
-                placeholder="https://"
-              />
-            </FormField>
-          </FormCard>
+          {application.isThirdParty && (
+            <FormCard
+              title="application_details.branding.more_info"
+              description="application_details.branding.more_info_description"
+            >
+              <FormField title="application_details.branding.terms_of_use_url">
+                <TextInput
+                  {...register('termsOfUseUrl', {
+                    validate: (value) =>
+                      !value || uriValidator(value) || t('errors.invalid_uri_format'),
+                  })}
+                  error={errors.termsOfUseUrl?.message}
+                  placeholder="https://"
+                />
+              </FormField>
+              <FormField title="application_details.branding.privacy_policy_url">
+                <TextInput
+                  {...register('privacyPolicyUrl', {
+                    validate: (value) =>
+                      !value || uriValidator(value) || t('errors.invalid_uri_format'),
+                  })}
+                  error={errors.privacyPolicyUrl?.message}
+                  placeholder="https://"
+                />
+              </FormField>
+            </FormCard>
+          )}
```

Comment:
Am I missing something? It seems only third-party apps can update terms of use and privacy policy URLs now?

Reply:
yes, it's by design at the moment

Reply:
ok